src/senaite/api/__init__.py

```diff
@@ -482,8 +482,11 @@ def get_path(brain_or_object):
     :rtype: string
     """
     if is_brain(brain_or_object):
-        return brain_or_object.getPath()
-    return "/".join(get_object(brain_or_object).getPhysicalPath())
+        path =  brain_or_object.getPath()
+        portal_path = get_path(get_portal())
+        if portal_path not in path:
+            return "{}/{}".format(portal_path, path)
+    return "/".join(brain_or_object.getPhysicalPath())
```

Comment:
I think you are missing here the `get_object(brain_or_object)`.
Also you don't handle here the case `if the portal_path in path`, which I assume is the case with a non uid_catalog brain.

Also the tests are failing for that one.